src/GusApi/Type/SearchParameters.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace GusApi\Type;
+
+class SearchParameters
+{
+    /**
+     * @var string
+     */
+    protected $Krs = null;
```

Comment:
`null` to zawsze domyślna wartość.

Reply:
Podobnie jak w innych klasach tutaj trzeba to wszystko wywalić bo automat generować to w jakiś taki dziwny sposób :/